pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -1039,6 +1039,13 @@ let
     } // optionalAttrs (versionAtLeast version "5.4" && stdenv.hostPlatform.system == "x86_64-linux") {
       CHROMEOS_LAPTOP = module;
       CHROMEOS_PSTORE = module;
+    } // optionalAttrs (versionAtLeast version "6.3" && (stdenv.hostPlatform.system == "x86_64-linux" || stdenv.hostPlatform.system == "aarch64-linux")) {
+      GCC_PLUGINS = no;
+      RUST = yes;
+      SAMPLES = yes;
+      SAMPLES_RUST = yes;
+      SAMPLE_RUST_MINIMAL = module;
+      SAMPLE_RUST_PRINT = module;
```

Comment:
Why build samples?

Reply:
As a test that it works I guess?

Reply:
Considering that they're going to end up in every kernel, I'd rather not.  It won't be long until there are real Rust drivers to test that it works.

Reply:
Have you checked whether this actually builds the samples? I don't see any Rust code being built?

Reply:
@yu-re-ka Did you have a moment to look into this?